cli/cmd/root.go

```diff
@@ -59,39 +59,41 @@ var rootCmd = &cobra.Command{
 			color.NoColor = noColor
 		}
 
-		err = util.ValidateEnvironment(context.Background())
+		detached, err := cmd.Flags().GetBool(flagDetached)
```

Comment:
is this related to validating if the image exists? I don't know if this is the right place for it. Also, if you move this code, can you pass in the flag values rather than using the global values.

Reply:
This is the right place for this code; none of these flags are accessible in `Execute()` context.

Reply:
o duh nevermind. I misread where this was